projects/hslayers/src/common/layer-shifting/layer-shifting.service.ts

```diff
@@ -1,123 +1,78 @@
-import {HsConfig} from '../../../config.service';
-import {HsEventBusService} from '../../core/event-bus.service';
-import {HsLayerManagerService} from '../layermanager.service';
-import {HsMapService} from '../../map/map.service';
-import {HsUtilsService} from '../../utils/utils.service';
 import {Injectable} from '@angular/core';
+
 import {Layer} from 'ol/layer';
 import {Source} from 'ol/source';
-import {getBase} from '../../../common/layer-extensions';
 
-export class PhysicalListItem {
+import {HsEventBusService} from '../../components/core/event-bus.service';
+import {HsLayerManagerService} from '../../components/layermanager/layermanager.service';
+import {HsMapService} from '../../components/map/map.service';
+import {HsUtilsService} from '../../components/utils/utils.service';
+import {getBase} from '../layer-extensions';
+
+export class LayerListItem {
   title: string;
-  layer: Layer<Source>;
+  layer: any;
```

Comment:
You can specify another type after  `|` instead of using any

Reply:
Couldn't use layer source because of the on listeners inside the swipecontrol class
.

Reply:
What about using `any` there?

Reply:
@raitisbe Added change for the type to be Layer<Source>, but made layer to be (as any) inside the swipeControl class.